tests/providers/jenkins/operators/test_jenkins_job_trigger.py

```diff
@@ -20,15 +20,23 @@
 
 import jenkins
 import mock
+from parameterized import parameterized
 
 from airflow.exceptions import AirflowException
 from airflow.providers.jenkins.hooks.jenkins import JenkinsHook
 from airflow.providers.jenkins.operators.jenkins_job_trigger import JenkinsJobTriggerOperator
 
+TEST_PARAMS = [
+    ({'a_param': 'blip', 'another_param': '42'},),
+    ('{"a_param": "blip", "another_param": "42"}',),
+    (['a_param', 'blip', 'another_param', '42'],),
+]
+
 
 class TestJenkinsOperator(unittest.TestCase):
+    @parameterized.expand(TEST_PARAMS)
```

Comment:
I personally prefer when the dict is placed in the decorator - no need to look for test cases :)